src/v2/Components/v2/ArtworkFilter/ArtworkFilterContext.tsx

```diff
@@ -196,11 +203,11 @@ export const ArtworkFilterContextProvider: React.FC<SharedArtworkFilterContextPr
     ZeroState,
 
     // Filter manipulation
-    isDefaultValue: field => {
+    isDefaultValue: (field) => {
```

Comment:
I think this needs a rebase from master to avoid the prettier churn?

Reply:
I tried running prettier in master and about a billion files got changed, so I guess we can keep that change here and do the whole project in another PR 🤔

Reply:
With prettier v2 we had to set this to `avoid` for consistency with the rest of the codebase: https://github.com/artsy/force/blob/master/package.json#L403. Wondering why your setup isn't picking up this setting? No harm in keeping it here, was more curious about if your environment is set up correctly. 

Def plan on running over the whole codebase with Prettier v2 defaults but there's a lot of open PRs right now, was gonna wait tilil things are less noisy.

Reply:
oh? hm I'll check the setup tomorrow.. interesting 🤔

Reply:
Hey @damassi @pvinis I'm going to go ahead and merge this as-is, in order to unblock #5658 — some of this churn will be undone in that PR in any case. 

Thanks in advance for taking a look at the setup tomorrow Pavlos.